packages/garbo/src/tasks/freeFight.ts

```diff
@@ -801,6 +802,34 @@ const FreeFightTasks: GarboFreeFightTask[] = [
   // tied-up leaviathan (scaling, has 100 damage source cap and 2500 hp)
   // li'l ninja costume
   // closed-circuit pay phone (make into it's own Quest)
+  {
+    name: "CyberRealm Overclock Fights",
+    ready: () =>
+      have($item`server room key`) && // TODO support day pass? Not require tanktop
+      have($item`zero-trust tanktop`) &&
+      have($skill`Torso Awareness`) &&
```

Comment:
I know some (unsupported) avatar paths have shirt access through another skill. Would it make more sense to use `canEquip`? ("No" is a perfectly fine answer here, I'm just spintballing)